ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusMessageReply.qml

```diff
@@ -111,14 +111,34 @@ Item {
                     asynchronous: true
                     active: replyDetails.contentType === StatusMessage.ContentType.Image
                     visible: active
-                    sourceComponent: StatusImageMessage {
+                    sourceComponent: replyDetails.albumCount > 0 ? albumComp : imageComponent
```

Comment:
Is there a way to just always use `albumComp`? Can it display an album with just one image in it? Or it's for backwards compatibility?

Reply:
Yeah, `Repeater` of `1` should yield the same result :)

Reply:
I will try :) `Repeater` of `1` without `album` shouldn't work @caybro  ;)

Reply:
What prevents you from doing `property var album: [singleImage]`? :)